dev/bots/test.dart

```diff
@@ -816,8 +816,10 @@ Future<void> _runWebLongRunningTests() async {
     () => _runGalleryE2eWebTest('profile', canvasKit: true),
     () => _runGalleryE2eWebTest('release'),
     () => _runGalleryE2eWebTest('release', canvasKit: true),
-  ].map(_withChromeDriver).toList();
+  ];
+  await _ensureChromeDriverIsRunning();
   await _selectIndexedSubshard(tests, kWebLongRunningTestShardCount);
```

Comment:
Suggestion:Shall we add a comment explaining why we shutdown?

Reply:
I'm not sure? All our tests try to shutdown processes they start. It's like `tearDown`. If you can think of something that would explain something special about chromedriver, I'd be happy to add it. LMK.